lambo

```diff
@@ -77,7 +77,10 @@ AUTH=false
 NODE=false
 CODEEDITOR=""
 BROWSER=""
-LINK=false' > ~/.lambo/config
+LINK=false
+CREATEDATABASE=false
+DBUSER=root
+BDPASSWORD=' > ~/.lambo/config
```

Comment:
DBPASSWORD, not BD

Reply:
done! language derived typo DataBase vd. Base de Dados 😅